src/components/Kv/KvVerificationCodeInput.vue

```diff
@@ -89,9 +89,15 @@ export default {
 	&__input {
```

Comment:
Because this component is set in `ems`, you can resize the whole deally by setting the font size.  That way the padding, letter-spacing etc. don't need to change.

In your AppAuthentication.vue it'd look something like
```
<kv-verification-code-input class="myclass">

.myclass {
  font-size: .75rem;

  @include breakpoint(large) {
    font-size: 1rem;
  }
}
```

Reply:
@ryan In this same PR, I'm making those changes inside AppAuthentication.vue. It's my understanding that the additional changes were needed inside KvVerificationCodeInput.vue because it doesn't fit on small screen. This is a screen shot of how the lightbox loads with just your suggested changes and KvVerificationCodeInput.vue reverted to how it was before. 
![Screen Shot 2021-01-15 at 9 38 42 AM](https://user-images.githubusercontent.com/1521381/104760975-097d4100-5717-11eb-91e3-c18152af8f77.png)
(*The input flows off the screen and the padding is too large to work on small screens)